bciers/libs/components/src/form/components/MultiStepButtons.tsx

```diff
@@ -61,33 +64,29 @@ const SubmitButton: React.FunctionComponent<SubmitButtonProps> = ({
           ))}
         {/* When the form is editable, the form should be submitted when navigating between steps */}
         {/* When the form is not editable (e.g., IRC staff is reviewing an operation), the form should not be submitted when navigating between steps */}
-        {!isFinalStep && disabled ? (
+        {!isFinalStep && disabled && !isSubmitting ? (
```

Comment:
I noticed this bug while working on this. This link would render while `isSubmitting` since the form gets disabled. To reproduce the bug: 
- checkout develop or any other branch
- go to page 1 of Operation Registration form (since it has uploads and takes awhile)
- fill it out form
- click `Save and Continue`
- click greyed out `Save and Continue` while the form is submitting
- submission will not finish but you will be taken to the following page